### DISCUSSION TITLE
Raw Tailwind CSS vs DaisyUI

### DISCUSSION THREAD

**Discussion by tempestshad0w** [saadeghi/daisyui#2915] [Q&A] [answered]
I could use lots of beautiful pre-defined components in daisyUI to my React app.
But how can I customize other components not mentioned in daisyUI such as Calendar component with daisyUI?
Should I use raw tailwind css? I want to know which one I should take for better customization. Raw tailwind css or daisyUI?

**Comment by Vijaykv5** [comment, marked as answer]
Hi @tempestshad0w 
I believe that fo customizing some components that are not covered by daisyUI, like a calendar, using raw Tailwind CSS would be more versatile. 
While daisyUI offers beautiful pre-defined components, for unique customization needs, raw Tailwind CSS provides more control and flexibility. You can use its utility classes to make elements precisely according to your requirements.

You may look into this [docs](https://daisyui.com/blog/daisyui-vs-tailwindui/) for comparing b/w `Tailwind css` and  `Daisy UI`.

**Reply by tempestshad0w** [reply]
Thanks for your help.
